docs/api/puppeteer.frame.addstyletag_1.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: Frame.addStyleTag_1
```

Comment:
we will have to do something about the overriden methods in the docs eventually

Reply:
At some point yes...